.github/workflows/spack.yml

```
@@ -0,0 +1,36 @@
+name: Spack
+
+on: pull_request
+
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Spack
+      run: |
+        cd ..
+        git clone https://github.com/spack/spack
+        cd spack
+        [[ $(uname) == "Linux" ]] && sudo apt-get install -y gfortran
```

I think this might let the Linux build get further. I don't know if there is a more elegant and future-proof form of this though.

```suggestion
        [[ $(uname) == "Linux" ]] && sudo apt-get install -y gfortran gfortran-7 gfortran-8 gfortran-9 gfortran-10
```